core/services/ardupilot_manager/mavlink_proxy/MAVProxy.py

```diff
@@ -48,7 +48,13 @@ def binary_name() -> str:
 
     @staticmethod
     def _validate_endpoint(endpoint: Endpoint) -> bool:
-        return True
+        return endpoint.connection_type in [
+            EndpointType.Serial,
+            EndpointType.UDPServer,
+            EndpointType.UDPClient,
+            EndpointType.TCPServer,
+            EndpointType.TCPClient,
```

Comment:
Are we going to support File endpoints at all ?
This commit is unnecessary if you put 74122 before.

Reply:
This was more to make the validation explicitly. Imagine we add a new EndpointType, say, potato. We cannot assume prior that the potato type is supported by MavProxy.